### PR TITLE
Kamila/solo game next button

### PR DESCRIPTION
**Pull request by kamilababayeva**
This PR add "Next" button to the TypingGame. The round does not start automatically but by the click on the "Next" button.

Updates:
- add button to the TypingGame Layout
- add button functionality to the TypingGame
- separate playRound into endRound/startRound
- improve testing of TypingGameActivity/GameActivity

For future PR:
- handle errors when it is impossible to fetch songs/song fragment
- notify user about data fetching
- think of more robust solution to fetch list of songs

image

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt [hunk 1]
```diff
@@ -108,7 +123,7 @@ class TypingGameActivity : GameActivity() {
                     try {
                         val list = Song.listSong(task.await())
                         for (x: Song in list) {
-                            guess(x, findViewById(R.id.displayGuess), this@TypingGameActivity, gameManager )
+                            guess(x,findViewById(R.id.displayGuess),this@TypingGameActivity,gameManager)
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt:126]:
```suggestion
                            guess(x, findViewById(R.id.displayGuess), this@TypingGameActivity, gameManager)
```
Why having deleted the space. It is with this format everywhere in the code ?

**Review by MaximeZmt — COMMENTED** [review]
👍 LGTM, Just one comment above !

**Review by Tsathogguaa — APPROVED** [review]
Great work!

File: app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt (hunk 1 above)
Comment (kamilababayeva) [reply app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt:126]: I think each parameter was on different line before. When I was moving them into one line, the space got deleted... We should format the code... I can do that in next PR

**Review by MaximeZmt — APPROVED** [review]
Nice Work 👍 !

**Pull request merged**